src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -72,7 +87,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
             if ( from == null || to == null || actual == null)
                 throw new ArgumentException( "Cannot compare using a null reference", "actual" );
```

Comment:
@CharliePoole it's the same as Is.LessThanOrEqualTo(5) but it's the more general form. Consider for instance having pairs of values supplied by `TestCase` attributes- no sense having to rewrite your code to use InRange or LessThanOrEqualTo, etc. The endpoints could also be calculated from another source.